packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -833,3 +850,15 @@ void _validateUrlString(String url) {
     throw ArgumentError(e);
   }
 }
+
+/// Simple credentials wrapper for authorization requests
+class AuthInfo {
+  /// Creates a new AuthInfo
+  AuthInfo(this.username, this.password);
```

Comment:
Since these have the same type, it would be safer to use `required` named arguments rather than positional arguments.